### PR TITLE
Updated ReadMe 

### LINKED ISSUE #1363: ReadMe out of date
Update the readme to include the necessary steps to get the project running.

### PR DESCRIPTION
**Pull request by rjaiswal823**
Closes #1363. Updated Doenet ReadMe with a description of the project and steps to get started developing on Doenet.

### CODE REVIEW

**Comment by Lyanthropos** [comment]
Common issues:
- Can not find the `doenet_local` database
  - run `npm create:db` and then `npm reset:db`
- esbuild fails to install
  - move to a node version `17.4` or higher
- cypress does not run
  - run `npm install` on your local machine first
- dev container crashes due to cypress error
  - make sure you have placed all cypress related dependencies in the optional category with `--save-optional`
- course creation or other basic functions are not working after an update
  - run `npm reset:db`

File: README.md [hunk 1]
```diff
@@ -1,25 +1,75 @@
...
+Starts the MySQL database.
+
+### `npm run reset:db`
+
+Resets the MySQL database.
+Running this command often fixes minor database loading issues.
+
+### `npm run publish:db`
+
+<!-- Get info from Kevin/Emilio -->
```
Comment (Lyanthropos) [inline README.md:59]: What info are you looking for here?

File: README.md [hunk 2]
```diff
@@ -1,25 +1,75 @@
...
+1. Fork the DoenetTools repository.
+
+   To enable the v2 docker compose CLI on Linux, [install the docker-compose-plugin](https://docs.docker.com/compose/install/linux/).
+
+2. Open the repository and run `npm install` to install all modules used by Doenet.
+
+3. To start your local version of Doenet, run `npm start` and view it in the browser at http://localhost:80.
+
+- If you are working with databases, we recommend installing [Sequel Ace](https://sequel-ace.com/) on MacOS.
+<!-- Any recs for Windows/Linux? -->
```
Comment (Lyanthropos) [inline README.md:30]: @dqnykamp You use MySQL workbench right?

Comment (dqnykamp) [reply README.md:30]: That's right.

**Pull request merged**